esrally/mechanic/supplier.py

```diff
@@ -725,7 +726,8 @@ def _update(self, revision):
 
     @classmethod
     def is_commit_hash(cls, revision):
-        return revision != "latest" and revision != "current" and not revision.startswith("@")
```

Comment:
So we're calling `is_commit_hash` in `_supply_requirements`, did you test that this new version will not affect behavior?

Reply:
This change doesn't affect behaviour, but I did discover that the approach doesn't account for users specifying a tag name (e.g. `--revision=v7.12.0`).